### PR TITLE
tornado initial app and requirements

### PR DESCRIPTION
**Pull request by neurosnap**
I've personally never used PostgreSQL with Tornado.  The usual route for tornado is a nosql database such as mongodb via motor.  However, I did some quick `wrk` and `wrk2` tests on the application and everything seems to working properly.

### CODE REVIEW

File: benchmarks/tornado/application.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        (r'/agents_with_pool', AgentsPool),
+        (r'/index', Index),
+    ], debug=False)
+
+    http_server = HTTPServer(application)
+    http_server.bind(8000)
+    http_server.start(None)
+
+    application.db = momoko.Pool(
+        dsn='dbname=benchmarks user=asterisk password=asterisk host=127.0.01 port=5432',
```
Comment (GMLudo) [inline benchmarks/tornado/application.py]: You should have a host typo in address, shouldn't you ?

**Comment by neurosnap** [comment]
Rebased to fix host typo.

**Comment by neurosnap** [comment]
Any updates on this PR?  Anything I can do to help?

**Comment by GMLudo** [comment]
Hi @neurosnap, are you at PyCON-US ?
It should be easier to discuss together.

**Comment by neurosnap** [comment]
I wish!  Unfortunately not.

**Comment by GMLudo** [comment]
Ok, no problems.
I've did some tests with your pull request, I've changed the response because it wasn't a JSON content.
I'm not a Tornado expert, nevertheless, based on your example, it was less performant than API-Hour version.
Nevertheless, I don't want to continue anymore my personal benchmark suite, because now, API-Hour will participate to the next round of FrameworkBenchmarks: http://www.techempower.com/benchmarks/
Where they will test more than 120+ REST frameworks in several languages, it will be more interesting for everybody.
In this benchmark, Tornado is already present.
If it isn't a problem for you, I propose to close this pull request.

**Pull request closed without merging**